analysis_2.yml

```diff
@@ -0,0 +1,89 @@
+name: analysis_2
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - attrs=19.3.0=py_0
+  - beautifulsoup4=4.9.1=py37hc8dfbb8_0
+  - brotlipy=0.7.0=py37h4ab8f01_1000
+  - ca-certificates=2020.4.5.2=hecda079_0
+  - catalogue=1.0.0=py_0
+  - certifi=2020.4.5.2=py37hc8dfbb8_0
+  - cffi=1.14.0=py37ha419a9e_0
+  - chardet=3.0.4=py37hc8dfbb8_1006
+  - cryptography=2.9.2=py37h26f1ce3_0
+  - cssselect=1.1.0=py_0
+  - cymem=2.0.3=py37h1834ac0_2
+  - cython-blis=0.4.1=py37h8055547_1
+  - feedfinder2=0.0.4=py_0
+  - feedparser=5.2.1=pyh9f0ad1d_0
+  - freetype=2.10.2=hd328e21_0
+  - idna=2.9=py_1
+  - importlib-metadata=1.6.1=py37hc8dfbb8_0
+  - importlib_metadata=1.6.1=0
+  - intel-openmp=2020.0=166
+  - jieba3k=0.35.1=py37hc8dfbb8_1002
+  - jpeg=9d=he774522_0
+  - jsonschema=3.2.0=py37hc8dfbb8_1
+  - libblas=3.8.0=15_mkl
+  - libcblas=3.8.0=15_mkl
+  - libiconv=1.15=hfa6e2cd_1006
+  - liblapack=3.8.0=15_mkl
+  - libpng=1.6.37=hfe6a214_1
+  - libtiff=4.1.0=h885aae3_6
+  - libxml2=2.9.10=h5d81f1c_1
+  - libxslt=1.1.33=h579f668_1
+  - lxml=4.5.1=py37h8ba8a40_0
+  - lz4-c=1.9.2=h62dcd97_1
+  - mkl=2020.0=166
+  - murmurhash=1.0.0=py37h63f7a3c_0
+  - newspaper3k=0.2.8=py37_0
+  - nltk=3.4.4=py_0
+  - numpy=1.18.5=py37hae9e721_0
+  - olefile=0.46=py_0
+  - openssl=1.1.1g=he774522_0
+  - pandas=1.0.5=py37h3bbf574_0
+  - pillow=7.1.2=py37hc826c6e_0
+  - pip=20.1.1=py_1
+  - plac=0.9.6=py37_0
+  - preshed=3.0.2=py37h1834ac0_3
+  - pycparser=2.20=py_0
+  - pyopenssl=19.1.0=py_1
+  - pyrsistent=0.16.0=py37h8055547_0
+  - pysocks=1.7.1=py37hc8dfbb8_1
+  - python=3.7.6=cpython_h60c2a47_6
+  - python-dateutil=2.8.1=py_0
+  - python_abi=3.7=1_cp37m
+  - pytz=2020.1=pyh9f0ad1d_0
+  - pyyaml=5.3.1=py37h8055547_0
+  - requests=2.23.0=pyh8c360ce_2
+  - requests-file=1.5.1=pyh9f0ad1d_0
+  - setuptools=47.1.1=py37hc8dfbb8_0
+  - six=1.15.0=pyh9f0ad1d_0
+  - soupsieve=2.0.1=py37hc8dfbb8_0
+  - spacy=2.2.4=py37h1fb7aa8_1
+  - spacy-model-en_core_web_sm=2.2.5=py_0
+  - sqlite=3.30.1=hfa6e2cd_0
+  - srsly=1.0.2=py37h1834ac0_0
+  - thinc=7.4.0=py37heaa310e_2
+  - tk=8.6.10=hfa6e2cd_0
+  - tldextract=2.2.0=py_0
+  - tqdm=4.46.1=pyh9f0ad1d_0
+  - urllib3=1.25.9=py_0
+  - vc=14.1=h869be7e_1
+  - vs2015_runtime=14.16.27012=h30e32a0_2
+  - wasabi=0.6.0=py_0
+  - wheel=0.34.2=py_1
+  - win_inet_pton=1.1.0=py37_0
+  - wincertstore=0.2=py37_1003
+  - xz=5.2.5=h2fa13f4_0
+  - yaml=0.2.5=he774522_0
+  - zipp=3.1.0=py_0
+  - zlib=1.2.11=h2fa13f4_1006
+  - zstd=1.4.4=h9f78265_3
+  - pip:
+    - gender-guesser==0.4.0
+    - genderize==0.3.1
+    - mediacloud==3.12.0
+prefix: C:\Users\isabe\Anaconda\envs\analysis_2
```

Comment:
This is specific to your development environment, so it should probably be removed (or made relative?).  If it's important for running the project then we need to make it configurable somehow.